### PR TITLE
FAQ: block and object storage

### PR DESCRIPTION
**Pull request by brasmi**
Added FAQ for block and object storage

Closes: #233

### CODE REVIEW

File: content/en/cloud/block-object-storage.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Does Lambda GPU Cloud offer block or object storage?"
+type: docs
+aliases:
+- guides/use-persistent-storage
```
Comment (cbrownstein-lambda) [inline content/en/cloud/block-object-storage.md:5]: We don't need a [redirect](https://gohugo.io/content-management/urls/#aliases), so we don't need these lines.

```suggestion
```

File: content/en/cloud/block-object-storage.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+---
+Lambda GPU Cloud currently doesn't offer block or object storage.
+
+Persistent storage needs to be created [using the dashboard](https://docs.lambdalabs.com/cloud/cloud-dashboard/#create-and-manage-persistent-storage-file-systems) and attached when
+you [create an instance](https://docs.lambdalabs.com/cloud/cloud-dashboard/#launch-restart-and-terminate-instances).
```
Comment (cbrownstein-lambda) [inline content/en/cloud/block-object-storage.md:10]: You want to use the [relref shortcode](https://gohugo.io/content-management/cross-references/) when linking to internal (docs site) pages. I'll explain why next time we meet.

Also, in most cases for readability, it's best to keep lines no longer than ~78 characters in length.

```suggestion
[using the dashboard]({{< relref "cloud-dashboard/#create-and-manage-persistent-storage-file-systems" >}})                                                                                                                                                    
and attached when you                                                                                                                                                                                                                                         
[create an instance]({{< relref "cloud-dashboard/#launch-restart-and-terminate-instances" >}}).  
```

File: content/en/cloud/block-object-storage.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (cbrownstein-lambda) [inline content/en/cloud/block-object-storage.md:11]: This whitespace (empty line) isn't needed.

```suggestion
```

**Comment by brasmi** [comment]
Applied feedback from review comments to block and object storage FAQ doc.

**Comment by cbrownstein-lambda** [comment]
My [suggestion](https://github.com/LambdaLabs/lambda-docs/pull/250#discussion_r1320393150), above, was crap and:

- Removed text that shouldn't have been removed.
- Added whitespace that shouldn't have been added.

In [6690e3e](https://github.com/LambdaLabs/lambda-docs/pull/250/commits/6690e3e1744802986b2af2c41f528beee1688665) I fixed the bugs introduced by my suggestion.

@brasmi The PR looks good now and I'll merge it momentarily.

Thank you for the great work!

**Pull request merged**